tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -382,6 +401,8 @@ const exportContacts = async () => {
   await (await exportButton()).click();
 };
 
+const cardFieldLabelText = async (label) => await (await $(`.cell.${label} label`)).getText();
```

Comment:
Can you please review [the comment from @latin-panda](https://github.com/medic/cht-core/pull/8054/files#r1107969512) in the previous review?